src/main/java/com/greenfox/lvlup/model/entity/Review.java

```diff
@@ -20,6 +20,8 @@
   private boolean pitchStatus;
   @ManyToOne
   private User user;
+
+  @JsonIgnore
```

Comment:
do you need this jsonignore for your codes? because it was removed previously by us, we don't need it, we use proper dto-s instead. if you don't need, can you remove it please?